pkg/scheduler/actions/preempt/preempt.go

```diff
@@ -117,8 +117,8 @@ func (pmpt *Action) Execute(ssn *framework.Session) {
 					if !api.PreemptableStatus(task.Status) {
 						return false
 					}
-					// Ignore task with empty resource request.
-					if task.Resreq.IsEmpty() {
+					// BestEffort pod is not supported to preempt unBestEffort pod.
```

Comment:
If a besteffort pod has a higher priority, preemption will not happen, is this expected?

Reply:
> If a besteffort pod has a higher priority, preemption will not happen, is this expected?

yes,  besteffort pod will be scheduled in "backfill" action, unbesteffort pods will be scheduled in "allocate" action. In normal use, allocate is executed before backfill, the preempted pod will be scheduled first in the next scheduler cycle